src/AutoBus.ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+import LocalTransport, { Listener, Transport } from './local/LocalTransport';
+
+export interface NextFunc<T> {
+  (error: Error | null, message: T): void;
+}
+
+export interface Middleware<T> {
+  (message: T, next: NextFunc<T>): void;
+}
+
+
+export default class AutoBus {
```

Comment:
I think the linter or formatter should prevent this? Remove duplicate blank line.

```suggestion
}

export default class AutoBus {
```